lib/committee/schema_validator/option.rb

```diff
@@ -46,6 +47,7 @@ def initialize(options, schema, schema_type)
         @optimistic_json                  = options.fetch(:optimistic_json, false)
         @parse_response_by_content_type   = options.fetch(:parse_response_by_content_type, true)
         @parameter_overwite_by_rails_rule = options.fetch(:parameter_overwite_by_rails_rule, true)
+        @permit_blank_structures          = options.fetch(:permit_blank_structures, false)
```

Comment:
Could you add a description of this parameter to the README?

https://github.com/interagent/committee?tab=readme-ov-file#configuration-options

Reply:
I have added a note to the README : )

[memo]

[In OpenAPI 3, the following becomes nil](https://github.com/ota42y/openapi_parser/blob/a430896dc0daa3b817e37e5c69791cf1841fb45b/lib/openapi_parser/schemas/response.rb#L26), so the validation of empty response succeeds.
Therefore, this option is expected to be valid only for Hyper-schema and OpenAPI 2.